SDALib-CPP/SDALib-Python/DriverTest.py

```diff
@@ -0,0 +1,30 @@
+import SDATypes
```

Comment:
Why is DriverTest in the SDALib-Python folder and not in the test folder?

Reply:
Because I didn't get CMake to work

Reply:
Because it needs SDATypes.py